manifests/base/values.yaml

```diff
@@ -11,3 +11,38 @@ global:
   # It is assumed that istio-system is running either 1.0 or an upgraded version of 1.1, but only security components are
   # used (citadel generating the secrets).
   istioNamespace: istio-system
+
```

Comment:
Why ? Are they used in the base templates ? We should only include what is required and used.

Reply:
yeah I just moved them in this PR. I think it's not that clear in the git diff

Reply:
See my other comments.

We should make it more clear that the 'base' is not intended to be customizable at this level, i.e. it should be possible to generate a plain yaml and just apply it without any kustomize or options.

It certainly shouldn't have revisions or in-place upgrades - like enabling telemetry on the global level would require. We can't revision the base easily.